### DISCUSSION TITLE
Why is the plugin showing the text as strikethrough in the Description Info page in Obsidian?

### DISCUSSION THREAD

**Discussion by itoldusoandso** [trganda/obsidian-attachment-management#114] [General]
See screenshot. 
It's all strike-through text in the plugin description.

I thought it is supposed to mean the functionality is broken right now. 

But it seems it is working for me at least.
![attman](https://github.com/trganda/obsidian-attachment-management/assets/71971273/244906d1-2959-43c9-8a4a-76f64d570cbe)

**Comment by luckman212** [comment]
It means that feature is completed. It's just a style, like when you cross something off your to-do list (literally)
